woocommerce/woocommerce-android
# Fix a crash on custom fields view

### Description

This PR fixes a crash specifically on tablet devices when the custom fields screen is visited from order detail screen.

### Testing instructions

Navigate to the order detail screen for any paid order and click on "view custom fields" CTA on tablet device and ensure there is no crash.

### Images/gif
